frontend/packages/integration-tests-cypress/tests/crud/other-routes.spec.ts

```diff
@@ -17,13 +17,14 @@ describe('Visiting other routes', () => {
   });
 
   const otherRoutes: { path: string; waitFor: () => void }[] = [
-    {
-      path: '/',
-      waitFor: () => {
-        cy.byLegacyTestID('resource-title').should('exist');
-        cy.byTestID('skeleton-chart').should('not.exist');
-      },
-    },
+    // disabled as <ServiceLevel> can return <Skeleton />, resulting in a a11y failure
```

Comment:
Is `Skeleton` a PF component or ours?

You could change the `waitFor` to check that skeleton doesn't exist.

Reply:
> Is `Skeleton` a PF component or ours?

Is a PatternFly component.
 
> You could change the `waitFor` to check that skeleton doesn't exist.

Good point.  I've updated accordingly.